[WIP]2016-07-18-july-2016-economical-software-testing.md

```diff
@@ -0,0 +1,53 @@
+---
```

Comment:
NOTE: File needs to end in `.md` to render as markdown and be viewable here. I made the change already directly to this PR.